ideas42ventures/ideas42ventures.com
# Adds submission-created Netlify function

Works on #80 

- Adds a serverless Netlify function.
- It checks for valid emails for our subscription form.


**How to test 🧪**
1. Click [here](https://deploy-preview-81--ideas42ventures.netlify.app/).
2. At the bottom of the page there's an email input field. 
3. Test it out by:
Invalid email - **2@2 SHOULD NOT** let you submit the form.
Valid email - **test@gmail.com SHOULD** let you submit the form.

Your should be able to see the valid email's [here](https://app.netlify.com/sites/ideas42ventures/forms/5f7472173932b50008579f97).
